dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/remove-fees-from-payment-group/remove-fees-from-payment-group.event-handler.ts

```diff
@@ -4,21 +4,24 @@ import { BaseUtilisationReportEvent } from '../../event/base-utilisation-report.
 import { FeeRecordStateMachine } from '../../../fee-record/fee-record.state-machine';
 import { feeRecordsMatchAttachedPayments } from '../helpers';
 
-type RemoveFeesFromPaymentEventPayload = {
+type RemoveFeesFromPaymentGroupEventPayload = {
   transactionEntityManager: EntityManager;
-  selectedFeeRecords: FeeRecordEntity[];
-  otherFeeRecords: FeeRecordEntity[];
+  feeRecordsToRemove: FeeRecordEntity[];
+  feeRecordsToUpdate: FeeRecordEntity[];
```

Comment:
I think this would be better named `otherFeeRecordsInGroup` because it allows us to understand that between the two variables we should have the full list of fee records within the group.

Reply:
Changed in edb8fe89b58fac28697fcc84bf11196ee74b1a42